README.md

```diff
@@ -1,189 +1,105 @@
-**Note** Information for plugin developers is [at the end of this document](#plugin-development).
+# Grafana data source plugin template
 
-This page describes the sumologic-metrics-grafana-datasource plugin, a datasource plugin for Grafana that can deliver metrics from your Sumo deployment to Grafana. After installing and configuring the plugin, you can query and visualize metrics from Sumo in the Grafana user interface. You initiate a query from the Grafana UI, the search runs on Sumo, and results are returned to Grafana.
+This template is a starting point for building a Data Source Plugin for Grafana.
 
-The Grafana backend will proxy all requests from the browser, and send them on to the data source.
+## What are Grafana data source plugins?
 
-This beta version of sumologic-metrics-grafana-datasource contains most planned features, but is not yet complete.
+Grafana supports a wide range of data sources, including Prometheus, MySQL, and even Datadog. There’s a good chance you can already visualize metrics from the systems you have set up. In some cases, though, you already have an in-house metrics solution that you’d like to add to your Grafana dashboards. Grafana Data Source Plugins enables integrating such solutions with Grafana.
```

Comment:
You have a typo - should be "and even Sumo Logic"

Reply:
it's generated :)